scripts/db_setup.sql

```diff
@@ -29,5 +29,6 @@ CREATE TABLE IF NOT EXISTS friends (
   username varchar(100) NOT NULL,
   friend_display_name varchar(20) NOT NULL,
   friend_mac_address varchar(18) NOT NULL,
-  friend_ip_address varchar(18) NOT NULL
+  friend_ip_address varchar(18) NOT NULL,
+  friend_username varchar(30) NOT NULL

```

Comment:
Why is friend_username a varchar(30) but the user's username a varchar(100)?

Reply:
That's a design decision that needs to be made - those are arbitrary currently and will be modified once these values are decided upon.